kata06/app/controllers/employees_controller.rb

```diff
@@ -0,0 +1,35 @@
+class EmployeesController < ApplicationController
+    def index
+        @employees = Employee.all
+    end
+    def new
+        @employee = Employee.new
+    end
+
+    def create
+        data = params.require(:employee).permit(:name, :email, :posicion, :num_empleado, :private_num)
+        Employee.create(data)
```

Comment:
you can also use the save method and compare the returned value to render a successful or error message

Reply:
ok! understood c: